include/nbl/video/ILogicalDevice.h

```diff
@@ -104,22 +104,11 @@ class NBL_API2 ILogicalDevice : public core::IReferenceCounted, public IDeviceMe
                 // https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkBufferMemoryBarrier2-srcStageMask-03851
                 // https://registry.khronos.org/vulkan/specs/1.3-extensions/html/vkspec.html#VUID-VkImageMemoryBarrier2-srcStageMask-03854
                 constexpr auto HostBit = asset::PIPELINE_STAGE_FLAGS::HOST_BIT;
-                if (barrier.dep.srcStageMask.hasFlags(HostBit)||barrier.dep.dstStageMask.hasFlags(HostBit))
+                if (barrier.dep.srcStageMask.hasFlags(HostBit) || barrier.dep.dstStageMask.hasFlags(HostBit)) {
```

Comment:
If you see our codebase, for multi-line control blocks (if/else/while/switch), position for open braces is usually on the next line.
I personally don't rely on and disabled auto-formatting, but if you're using one. Select the positiin of open braces to be on a `new line`
![image](https://github.com/user-attachments/assets/db9a3020-2f62-4454-8405-481a19aafdb3)